### PR TITLE
Symfony5

### PR DESCRIPTION
**Pull request by benji07**
## Todo

- [x] Need https://github.com/phpspec/phpspec/pull/1276

### CODE REVIEW

File: spec/Service/RefreshTokenSpec.php [hunk 1]
```diff
@@ -23,6 +23,8 @@ public function let(RefreshTokenAuthenticator $authenticator, RefreshTokenProvid
         $ttlUpdate = false;
         $providerKey = 'testkey';
 
+        $eventDispatcher->dispatch(Argument::any(), Argument::any())->willReturn(Argument::any());
```
Comment (benji07) [inline spec/Service/RefreshTokenSpec.php]: i don't understand why i need to add this line in the test for them to pass

Comment (benji07) [reply spec/Service/RefreshTokenSpec.php]: If i don't have this line test fail fait:

```
Gesdinet/JWTRefreshTokenBundle/Service/RefreshToken                                
  34  - it refresh token
      exception [err:TypeError("Return value of Double\EventDispatcherInterface\P110::dispatch() must be an object, null returned")] has been thrown.
```

**Comment by mathroc** [comment]
phpspec/phpspec#1276 has been merged and released

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The purpose of this bundle is manage refresh tokens with JWT (Json Web Tokens) i
 Prerequisites
 -------------
 
-This bundle requires Symfony 3.3+ or 4.0+.
+This bundle requires Symfony 3.3+, 4.0+ or 5.0+.
```
Comment (norkunas) [inline README.md]:
```suggestion
This bundle requires Symfony 3.4+, 4.0+ or 5.0+.
```

**Comment by tidall87** [comment]
Please someone for merge?

**Pull request merged**